scripts/webpack/webpack.dev.js

```diff
@@ -32,7 +32,15 @@ module.exports = (env = {}) => {
 
     // If we enabled watch option via CLI
     watchOptions: {
-      ignored: /node_modules/,
+      ignored: [
+        '/node_modules/',
+        '**/plugins/datasource/azuremonitor/**',
+        '**/plugins/datasource/cloud-monitoring/**',
+        '**/plugins/datasource/grafana-pyroscope-datasource/**',
+        '**/plugins/datasource/grafana-testdata-datasource/**',
+        '**/plugins/datasource/parca/**',
+        '**/plugins/datasource/tempo/**',
+      ],
```

Comment:
this list seems incomplete (compared to https://github.com/grafana/grafana/blob/main/.eslintrc#L98-L119). Maybe we can read / adapt that list here?

Reply:
I'll need to dig into what's going on in that eslintrc. I only consider a decoupled datasource when it has a package.json and according to my IDE if I'm on `main` I only see the following. 🤔 

![image](https://github.com/grafana/grafana/assets/73201/39a09f79-3a3d-44d9-afb4-5510205f125a)

Reply:
yeah, plugins can have the linter rules active but still not be fully decoupled. Then, can we add some logic here to find package.jsons? I am trying to avoid a manual step for people that will likely forget.

Reply:
Nice suggestion @andresmgot ! I've added something in https://github.com/grafana/grafana/pull/81873/commits/7584e613631abc86941708ee91ebe246fde5a224 which should address this without the need for additional steps during decoupling.

Reply:
:clap: